docs/conf/atlantic/2024/unconference.rst

```diff
@@ -33,29 +33,29 @@ To lead an unconf session, propose a topic and pick a time; unconf sessions are
 
 {% if unconf and unconf.url %}
 
-Sessions take place in Hopin, but you propose a session by adding your topic to a timeslot and table number in the  `Unconference sign-up sheet <{{unconf.url}}>`__.
+Sessions take place in the virtual conference platform, but you propose a session by adding your topic to a timeslot and table number in the  `Unconference sign-up sheet <{{unconf.url}}>`__.
```

Comment:
Looks good. We should probably add a virtual conf platform variable?

Reply:
There aren't that many places, except the welcome wagon guide, which will need a rewrite anyways